src/piltover_input.cairo

```diff
@@ -0,0 +1,77 @@
+use piltover::snos_output::{StarknetOsOutput, deserialize_os_output};
+
+pub const HEADER_SIZE: usize = 8;
+
+#[derive(Drop, Serde, Debug)]
+pub struct LayoutBridgeOutput {
+    pub bootloader_task_count: felt252,
+    pub output_length: felt252,
+    pub layout_bridge_program_hash: felt252,
+    pub bootloader_program_hash: felt252,
+    pub snos_output_length: felt252,
+    pub bootloader_output: SnosBootloaderOutput,
+}
+
+
+#[derive(Drop, Serde, Debug)]
+pub struct SnosBootloaderOutput {
+    pub bootloader_task_count: felt252,
+    pub output_length: felt252,
+    pub snos_program_hash: felt252,
+    pub snos_output: StarknetOsOutput,
+}
+
+pub fn deserialize_layout_bridge_output(data: Span<felt252>) -> LayoutBridgeOutput {
+    let snos_output: Array<felt252> = data
+        .slice(HEADER_SIZE, data.len() - HEADER_SIZE)
+        .into_iter()
+        .map(|f| *f)
+        .collect();
+    let mut snos_output_iter = snos_output.span().into_iter();
+    let output = deserialize_os_output(ref snos_output_iter);
+
+    LayoutBridgeOutput {
```

Comment:
Would be good to have a link to somewhere that link to Cairo code or some documentation around those values. Just to contextualize the reader.

Wondering what's the approach we should use though. If the layout bridge program is somewhere in the sequencer/starkware repo, let's link it.